server/routes/api/disaster_api.py

```diff
@@ -49,21 +50,134 @@ def event_date_range():
   return Response(json.dumps(result), 200, mimetype='application/json')
 
 
+def keep_event(event, place, filter_prop, filter_unit, filter_upper_limit,
+               filter_lower_limit):
+  """
+  Returns whether or not we should keep an event
+  """
+  if place != EARTH_DCID and not place in event.get("affectedPlaces", []):
+    # event does not affect the place of interest
+    return False
+  if not filter_prop or not filter_prop in event:
+    # no props to filter by or event doesn't contain the prop information
+    return True
+  prop_val = event[filter_prop]
+  if len(prop_val) < len(filter_unit):
+    # value for the property to filter by is empty
+    return True
```

Comment:
we should return false here?

Reply:
in mixer, if the prop isn't found or can't be parsed, we keep the event .. not sure if we want this logic to be different from mixer logic?